include/rsl/parameter_validators.hpp

```diff
@@ -218,7 +218,7 @@ template <typename T>
  * @return Help string if the parameter is invalid, otherwise void
  */
 template <typename T>
-[[nodiscard]] auto lower_bounds(rclcpp::Parameter const& parameter, T const& value) {
+[[deprecated, nodiscard]] auto lower_bounds(rclcpp::Parameter const& parameter, T const& value) {
```

Comment:
Do you mind adding a message to the deprecation telling users what to use instead?